app/services/bgs/dependent_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module BGS
+  class DependentService
+    def get_dependents(current_user)
+      service = LighthouseBGS::Services.new(
+        external_uid: current_user.icn,
+        external_key: current_user.email
+      )
+
+      service.claimants.find_dependents_by_participant_id(current_user.participant_id, current_user.ssn)
```

Comment:
What happens if this fails?

Reply:
There is a "rescue" in the controller that calls this method that will catch the error from the lighthouse_bgs service and log it to sentry.